README.md

```diff
@@ -274,6 +274,22 @@ If you have a `node_modules/` folder, it must be in the same directory as `contr
 run the compilation command for the framework you are using e.g `npx hardhat compile`. That must work successfully;
 otherwise, slither's compilation engine, crytic-compile, cannot generate the AST.
 
+* If `node_modules/` is not in your `contracts/` directory of your project, you will need to involve **remappings**. If a project has dependencies like OpenZeppelin or Chainlink, but you want to run Slither on a single `.sol` file or use [slither's tools](https://github.com/crytic/slither#tools) like `slither-check-erc`, you can add the `--solc-remaps @=node_modules/@` flag and/or configure `slither.config.json` in your project's root with the remapping. If `slither.config.json` remappings are setup the flag is no longer needed unless using [slither's tools](https://github.com/crytic/slither#tools). Foundry projects also require remappings if you want to analyze a specific `.sol` file that requires imports. Read more here: [Foundry Static Analysis Docs](https://book.getfoundry.sh/config/static-analyzers).
+
```

Comment:
>  Foundry projects also require remappings if you want to analyze a specific `.sol` file that requires imports. Read more here: [Foundry Static Analysis Docs](https://book.getfoundry.sh/config/static-analyzers).

I'm not sure this is accurate as of the 0.10.0 release. We use  Foundry (which automatically configures the remappings) to compile single files

Reply:
I'm not sure if the last bit ("... or use a [slither tool](https://github.com/crytic/slither#tools) like `slither-check-erc`...") is accurate either. You can run both `slither` and tools like `slither-check-erc` with a project directory (like `.`) as target and not worry about remappings; in the case of standalone tools, they take the contract name you want to analyze:

```
% slither-check-erc .  Counter
# Check Counter

## Check functions
[ ] totalSupply() is missing 
[ ] balanceOf(address) is missing 
[ ] transfer(address,uint256) is missing 
[ ] transferFrom(address,address,uint256) is missing
...
```

Reply:
elopez is correct my mistake, `slither-check-erc .  Counter` does work without any remappings.